src/test/scala/chiselTests/Vec.scala

```diff
@@ -371,100 +100,231 @@ class VecSpec extends ChiselPropSpec with Utils {
     }
   }
 
-  property("A Reg of a Vec should operate correctly") {
-    forAll(safeUIntN(8)) {
-      case (w: Int, v: List[Int]) =>
-        assertTesterPasses { new RegTester(w, v) }
-    }
-  }
-
-  property("A Vec of lit should operate correctly") {
-    forAll(safeUIntN(8)) {
-      case (w: Int, v: List[Int]) =>
-        assertTesterPasses { new LitTester(w, v) }
-    }
-  }
-
```

Comment:
This test actually didn't do anything, it passed values but they were ignored and it just used zero. I figure Vecs of Lits are better tested by the Vec literal stuff anyway.